provider/coredns/coredns.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
I think we should drop the init() func. The change also does not make sense to me.

Reply:
removed in my last commit